tardis/montecarlo/base.py

```diff
@@ -197,11 +197,17 @@ def run(self, model, plasma, no_of_packets,
         Parameters
         ----------
         model : tardis.model.Radial1DModel
+            # UNSURE of description of below params
```

Comment:
If you don't know about this, better leave it as it is - no need of comment. Similarly, remove `.` from following parameters.

Reply:
okay